velox/exec/tests/HashJoinTest.cpp

```diff
@@ -8112,4 +8112,85 @@ TEST_F(HashJoinTest, nanKeys) {
        makeFlatVector<int64_t>({1, 2})});
   facebook::velox::test::assertEqualVectors(expected, result);
 }
+
+TEST_F(HashJoinTest, combineSmallVectorsAfterFilter) {
+  // Verify low selectivity / small vectors are combined to 1 vector.
```

Comment:
Would you add a comment to describe the shape of the data required for this test?

Reply:
Updated, thanks.